featuretools/computational_backends/pandas_backend.py

```diff
@@ -84,9 +84,6 @@ def calculate_all_features(self, instance_ids, time_last,
 
         target_entity = self.entityset[self.target_eid]
 
-        if not target_entity.df.index.isin(instance_ids).any():
-            return self.generate_default_df(instance_ids=instance_ids)
```

Comment:
Should we leave this as an optimization, since we can catch this case early?

Reply:
I think it depends on the time necessary to perform this check. How does the run time change as the index grows / we add more missing instances?  This code will be run for each unique time in `cutoff_time`.  Quickly catching if every instance is not present in the data might not be worth optimizing for.

Reply:
Good point, I didn't think about the cost of the check itself. Since it should be a rare case it seems unlikely to be beneficial, and adds complexity.